undebt/cmd/main.py

```diff
@@ -57,21 +57,21 @@ def _write_result_text(result_text, path, dry_run):
 def _handle_arguments():
     parser = argparse.ArgumentParser(prog='undebt')
     parser.add_argument(
-        '--input', '-i', metavar='path', action='append',
```

Comment:
By [semver](http://semver.org/) we're still in 0.x territory, so I don't see why we have to be concerned with backwards compat in the CLI. Just bump the minor version